lamindb_setup/_schema_metadata.py

```diff
@@ -356,21 +356,23 @@ def to_json(self):
         return self.to_dict(include_django_objects=False)
 
     def _get_modules_metadata(self):
-        return {
+        all_models = {
             module_name: {
                 model._meta.model_name: _ModelHandler(
                     model, module_name, self.included_modules
                 )
                 for model in self._get_schema_module(
                     module_name
                 ).models.__dict__.values()
-                if model.__class__.__name__ == "RecordMeta"
-                and model.__name__ not in ["Record", "ORM"]
+                if model.__class__.__name__ == "Registry"
+                and model.__name__ != "Record"
                 and not model._meta.abstract
                 and model.__get_schema_name__() == module_name
             }
             for module_name in self.included_modules
         }
+        assert all_models
```

Comment:
@fredericenard @sunnyosun this is so that the problem from this morning doesn't happen again: it checks that there are at least some models in the schema

Reply:
But looking at this, it's somewhat bad code anyway to do this with strings... I'll refactor.
<img width="410" alt="image" src="https://github.com/user-attachments/assets/87108858-ac17-4e3c-8c0f-5d2b7fb5f4a3">

Reply:
@falexwolf I'm not sure this check is the right one, because the dictionary wasn't empty. 
What was empty is `all_models[module_name]`.

Reply:
I would suggest something like:

```
assert all(bool(module_dict) for module_dict in all_models.items())
```

Reply:
Sounds good, Fred! Can you amend the check?